hvac/tests/test_integration.py

```diff
@@ -116,6 +116,10 @@ def test_secret_backend_manipulation(self):
         self.client.enable_secret_backend('generic', mount_point='test')
         assert 'test/' in self.client.list_secret_backends()
 
+        self.client.tune_secret_backend('generic', mount_point='test', default_lease_ttl='3600s', max_lease_ttl='8600s')
+        assert 'max_lease_ttl' in self.client.get_secret_backend_tuning('generic', mount_point='test')
+        assert 'default_lease_ttl' in self.client.get_secret_backend_tuning('generic', mount_point='test')
```

Comment:
I think this needs to include a test that the tuning actually modified the `default_lease_ttl` and `max_lease_ttl` for the mount.

Reply:
Yes please. If we can get that test, we can merge.

Reply:
I branched off of this PR and added the additional tests in #137